apis/python/setup.py

```diff
@@ -269,7 +269,7 @@ def run(self):
     install_requires=[
         # Needed for Python 3.7 which anndata 0.9 doesn't support but we do
         "anndata < 0.9; python_version<'3.8'",
-        "anndata; python_version>='3.8'",
+        "anndata~=0.9.0; python_version>='3.8'",  # XXX TEMP
```

Comment:
anndata 0.10.0 just dropped and this is affecting ALL our Python PRs right now & I'll come up with a way to handle this -- got some good advice from @thetorpedodog in Slack ...

Reply:
Tracked in #1765